kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/dynamo/magmatic.js

```diff
@@ -0,0 +1,18 @@
+onEvent('recipes', (event) => {
+    var multiplier = 10;
+    var data = {
+        recipes: [
+            {
+                fluid: 'tconstruct:blazing_blood',
+                energy: 1000000
+            }
+        ]
+    };
+    data.recipes.forEach((recipe) => {
+        event.recipes.thermal.magmatic_fuel({
+            type: 'thermal.magmatic_fuel',
```

Comment:
does this work? seems like a bit of a mix between KubeJS Thermal and event.custom

Reply:
Oh yeah good point. These are all a little wonky. I think they predate the thermal kubejs add-on and I just copied. I'll clean em all up here